webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import {
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+
+const ContentViewVersionRepositoryCell = ({
+  data: {
+    content_counts: ContentCounts,
+    product: { id },
+    library_instance_id: libraryInstanceId,
+  },
+}) => {
+  const CONTENT_COUNTS = {
+    ansible_collection: {
+      name: __('Ansible collections'),
+      url: `ansible_collections?repositoryId=${libraryInstanceId}`,
+    },
+    deb: {
+      name: __('Deb packages'),
+      url: `debs?repositoryId=${libraryInstanceId}`,
+    },
+    docker_manifest: {
+      name: __('Docker manifests'),
+    },
+    docker_manifest_list: {
+      name: __('Docker manifest lists'),
```

Comment:
'Container manifest lists' for consistency on the page.
Also could you update it in the versions additional content component to use container over docker.
URL takes the form: products/1/repositories/72/content/content/docker_manifest_lists?page=1&per_page=20

Reply:
Thank you! Addressed all of the above